deno/fhirchat-demo/hooks/aiConversationContext.ts

```diff
@@ -1,4 +1,4 @@
-import { type Signal, signal } from "@preact/signals";
+import { signal, type Signal } from "@preact/signals";
```

Comment:
This was automatically switched using my formatter

Reply:
I get the same switch actually:)